nginx/generate-config

```diff
@@ -14,8 +14,10 @@ LOCATIONS=$(
 		[ "$name" == "restreamer" ] && generate_location / "http://restreamer:$port"
 		# thrimshim takes any calls to thrimshim/
 		[ "$name" == "thrimshim" ] && generate_location /thrimshim "http://thrimshim:$port"
-		# all services have metrics under /metrics/SERVICE
-		generate_location "/metrics/$name" "http://$name:$port/metrics"
+		# thrimbletrimmer takes any calls to thrimbletrimmer/; serves content from /etc/nginx/html/thrimbletrimmer; requests to the plain URL get redirect to the dashboard page.
+		[ "$name" == "thrimbletrimmer" ] && echo -e "\t\tlocation = / { return 301 \$scheme://\$host/thrimbletrimmer/dashboard.html; }\n\t\tlocation /thrimbletrimmer { }"
+		# all services have metrics under /metrics/SERVICE, except for thrimebletrimmer
+		[ "$name" != "thrimbletrimmer" ] && generate_location "/metrics/$name" "http://$name:$port/metrics"
```

Comment:
see my comment above, I think this can be done cleaner by not treating thrimbletrimmer as a service.